src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -337,3 +575,48 @@ class ListboxWithOptions {
     this.changedOption = event.option;
   }
 }
+
+@Component({
+  template: `
+    <div cdkListbox
+         [multiple]="isMultiselectable"
```

Comment:
There's an extra space here?

Reply:
Actually I think the `(selectionChange)` on the next line needs a space.